ale_linters/python/flake8.vim

```diff
@@ -38,10 +38,25 @@ function! ale_linters#python#flake8#RunWithVersionCheck(buffer) abort
     \)
 endfunction
 
+function! ale_linters#python#flake8#GetCdString(buffer) abort
+    let l:change_directory = ale#Var(a:buffer, 'python_flake8_change_directory')
+
+    if l:change_directory == 2
```

Comment:
We can do better than `2`. Let's add strings for the existing options and new option, and also continue to support `1` and `0`. Make sure to update the documentation, including the documented type.

Let's make this new setting the new default, and make sure we fall back on setting the CWD to the directory the file is in, in the Vader tests.